python/lib/eeg.py

```diff
@@ -816,4 +927,74 @@ def create_and_insert_archive(self, files_to_archive, archive_rel_name,
         }
         physiological.insert_archive_file(archive_info)
 
+    def create_and_insert_annotation_archive(self, files_to_archive, archive_rel_name,
+                                  eeg_file_id):
+        """
+        Create an archive with all annotations files associated to a specific recording
+
+        :param files_to_archive: tuple with the list of files to include in
+                                 the archive
+         :type files_to_archive: tuple
+        :param archive_rel_name: path to the archive relative to data_dir
+         :type archive_rel_name: str
+        :param eeg_file_id     : PhysiologicalFileID
+         :type eeg_file_id     : int
+        """
+
+        # load the Physiological object that will be used to insert the
+        # physiological archive into the database
+        physiological = Physiological(self.db, self.verbose)
+
+        # check if archive is on the filesystem
+        archive_full_path = self.data_dir + archive_rel_name
+        blake2            = None
+        if os.path.isfile(archive_full_path):
+            blake2 = blake2b(archive_full_path.encode('utf-8')).hexdigest()
+
+        # check if archive already inserted in database and matches the one
+        # on the filesystem using blake2b hash
+        query = "SELECT * " \
```

Comment:
the query should go in a function in `lib/database_lib/physiologicalannotationarchive.py` file where all queries to the physiological_annotation_archive table would be.